From 4262e671f950b5141db7c5899c1939fb07e94149 Mon Sep 17 00:00:00 2001
From: Algea Cao <algea.cao@rock-chips.com>
Date: Tue, 6 Feb 2018 09:26:14 +0800
Subject: [PATCH] arm64: dts: rk3368-r88: support rk3368 drm cvbs uboot logo

Removing ports so that uboot display subsystem can get
rk1000 as panel.

Change-Id: If12c30bb7d1bd382ed969534687234aa79b8dd04
Signed-off-by: Algea Cao <algea.cao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-r88.dts | 18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-r88.dts b/arch/arm64/boot/dts/rockchip/rk3368-r88.dts
index ccc09e7cf30c..61bc28c76115 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-r88.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-r88.dts
@@ -474,14 +474,12 @@
 		rockchip,data-width = <24>;
 		rockchip,output = "rgb";
 		rockchip,ctl = <&rk1000_ctl>;
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-			tve_in: port@0 {
-				reg = <0>;
-				tve_in_lvds: endpoint {
-					remote-endpoint = <&lvds_out_tve>;
-				};
+		#address-cells = <1>;
+		#size-cells = <0>;
+		tve_in: port@0 {
+			reg = <0>;
+			tve_in_lvds: endpoint {
+				remote-endpoint = <&lvds_out_tve>;
 			};
 		};
 	};
@@ -674,6 +672,10 @@
 	status = "okay";
 };
 
+&route_lvds {
+	status = "okay";
+};
+
 &hdmi {
 	status = "okay";
 	#sound-dai-cells = <0>;
-- 
2.35.3

